src/Aspire.Dashboard/Components/Controls/LogViewer.razor

```diff
@@ -6,7 +6,27 @@
 @inject IJSRuntime JS
 @implements IAsyncDisposable
 
-<div class="log-overflow">
+<div class="log-overflow continuous-scroll-overflow">
     <div class="log-container" id="logContainer">
+        <Virtualize Items="_logEntries" ItemSize="20" OverscanCount="20">
```

Comment:
Since the entries being added are fairly lightweight, we could probably go with an even higher OverscanCount for this to help with what I expect will be commonly occurring rapid scrolling. Not sure how to determine what would be a good number though. One full extra page on a reasonable size monitor?